### ISSUE TITLE
"Skip to main" link goes to repeated nav rather than main content

### ISSUE THREAD

**Issue by calumryan** [alphagov/govuk-design-guide#311]
On pages with the sidebar navigation before the main content, the ‘Skip to main content’ link does not link users to the perceivable main content of the page. Navigating repeated blocks of content such as navigation elements (e.g., ‘Pages in this section’) can be time-consuming and requires additional key-presses, which can become strenuous for users with mobility impairments.

Fails: WCAG 2.2 2.4.1 Bypass Blocks (Level A)
